src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests.cs

```diff
@@ -606,7 +604,7 @@ async Task EvaluateOnCallFrameFail(string call_frame_id, params (string expressi
                    ("f.textList[j]", TString("2")),
                    ("f.numArray[j]", TNumber(2)),
                    ("f.textArray[i]", TString("1")));
-                
+
```

Comment:
There are some accidentally added blank lines here.

Reply:
These are existing lines blank lines, that had trailing spaces. So, the diff makes them true blank lines :)